primefaces/src/main/java/org/primefaces/component/splitbutton/SplitButtonRenderer.java

```diff
@@ -64,6 +66,11 @@ public void decode(FacesContext context, UIComponent component) {
                 component.queueEvent(new ActionEvent(component));
             }
         }
+
+        OverlayPanel customOverlay = button.getCustomOverlay();
+        if (customOverlay != null) {
+            new OverlayPanelRenderer().decode(context, customOverlay);
```

Comment:
get the Renderer instance via RenderKit, a user can overwrite it via faces-config